docs/customer_management/cp_configuration.md

```diff
@@ -89,3 +89,9 @@ ibexa:
                         match:
                             Identifier\ContentType: confirmation_page
 ```
+
+# Order management
+
+Reviewing pending and past orders in Customer Portal requires that you configure all currencies that any of the customers may use under the `ibexa.system.default.product_catalog.currencies` key. 
```

Comment:
Why currencies list needs to be defined in `default` scope ?

Reply:
Good catch, it actually doesn't. We can change it to sth like:
`ibexa.system.<your_siteaccess>.product_catalog.currencies`.